src/main.js

```diff
@@ -69,23 +69,23 @@ fetch(data_location) // eslint-disable-line
 
     // pinia is a framework used for managing global state.
     const pinia = createPinia()
-        const app = createApp(App, {siteData: response}); // eslint-disable-line
+    const app = createApp(App, { siteData: response }) // eslint-disable-line
 
     app.use(capeRouter)
     app.use(pinia)
 
     // read data from local.js and populate the values into the environment store
     const environmentStore = useEnvironmentStore()
-        environmentStore.appStatus = app_status; // eslint-disable-line
-        environmentStore.buildId = build_id; // eslint-disable-line
+    environmentStore.appStatus = app_status // eslint-disable-line
+    environmentStore.buildId = build_id // eslint-disable-line
 
-    // eslint-disable-next-line no-undef
-    if (typeof cape_extensions !== 'undefined' && cape_extensions && Object.prototype.hasOwnProperty.call(cape_extensions, 'components')) {
-      // eslint-disable-next-line no-undef
-      const extensionIds = Object.keys(cape_extensions.components)
+    // add any vue extensions if defined. Remap the variable name for cleaner linting
+    const capeExtensions = cape_extensions // eslint-disable-line
```

Comment:
Should we change the name of variable `cape_extensions`  to `capeExtensions` instead of creating a new variable?

Reply:
Ah, no. This variable is set outside the modules in the browser itself using local.js
We probably will rename these when we do a new major version but it would mean changing the deployment pipelines as they are what make local.js

Reply:
Ah, that makes sense.